alonzo/impl/test/test/Test/Cardano/Ledger/Alonzo/Examples/Utxow.hs

```diff
@@ -142,19 +142,54 @@ initUTxO =
   UTxO $
     Map.fromList
       [ (TxIn genesisId 0, alwaysSucceedsOutput),
-        (TxIn genesisId 1, alwaysFailsOutput),
-        (TxIn genesisId 2, feeOutput)
+        (TxIn genesisId 1, feeOutput),
+        (TxIn genesisId 2, feeOutput),
+        (TxIn genesisId 3, feeOutput),
+        (TxIn genesisId 4, feeOutput),
+        (TxIn genesisId 5, feeOutput),
+        (TxIn genesisId 6, feeOutput),
+        (TxIn genesisId 7, feeOutput),
+        (TxIn genesisId 8, feeOutput),
+        (TxIn genesisId 9, alwaysFailsOutput)
       ]
 
 initUtxoSt :: UTxOState A
 initUtxoSt = UTxOState initUTxO (Coin 0) (Coin 0) def
 
+-- | This is a helper type for the expectedUTxO function.
+--  IsSpend indicates that we created a transaction which
+--  successfully spends an output locked by a Plutus script.
+data IsSpend = IsSpend | IsNotSpend
+
+-- | This is a helper type for the expectedUTxO function.
+--  ExpectSuccess indicates that we created a valid transaction
+--  where the IsValidating flag is true.
+data Expect = ExpectSuccess (TxBody A) (TxOut A) IsSpend | ExpectFailure
+
+-- | We always expect the alwaysFailsOutput to remain in the UTxO map.
+-- Each of the eight example transactions below will consume one of the
+-- feeInputs, given by feeInputs.
+-- The transaction where IsValidating is True will create a transaction output.
+-- Additionally, the alwaysFailsOutput will be consumed in the IsSpend case.
+expectedUTxO :: Expect -> Natural -> UTxO A
```

Comment:
Add to the comments how this should be used. The comments tells what it does, not how one should use it.